docs/en/models/yolov7.md

```diff
@@ -103,12 +103,14 @@ We would like to acknowledge the YOLOv7 authors for their significant contributi
     === "BibTeX"
 
         ```bibtex
+        {% raw %}
```

Comment:
Why has this been applied here? Why are the other 10 pages with bibtex entries fine without it?

Reply:
Actually looking closely the problem here is the citations in bibtex. These two pages had a wrong syntax. I regenerated the citations in bibtex with google scholar and the correct format is as follows (Without curly brackets around YOLOv7):
```bibtex
@article{wang2022yolov7,
  title={YOLOv7: Trainable bag-of-freebies sets new state-of-the-art for real-time object detectors},
  author={Wang, Chien-Yao and Bochkovskiy, Alexey and Liao, Hong-Yuan Mark},
  journal={arXiv preprint arXiv:2207.02696},
  year={2022}
  }}
```
Wrong:
```bibtex
@article{wang2022yolov7,
  title={{YOLOv7}: Trainable bag-of-freebies sets new state-of-the-art for real-time object detectors},
  author={Wang, Chien-Yao and Bochkovskiy, Alexey and Liao, Hong-Yuan Mark},
  journal={arXiv preprint arXiv:2207.02696},
  year={2022}
  }}
```

No need to add any raw text syntax here. Updated!